socorro.js

```diff
@@ -60,6 +66,7 @@ if (Object.keys(fileinfo).length != 0) {
       fetchCoverage(revision, filename).then(data => {
         if (data !== null && !data.hasOwnProperty("error")) {
           if (!data.hasOwnProperty("data")) {
+            removeSpinners(lineElements);
```

Comment:
Instead of calling removeSpinners here and in the successful case, you could use `finally` on the promise returned by fetchCoverage.then:
```
fetchCoverage(revision, filename)
.then(data => {
    ...
})
.finally(() => {
  // REMOVE SPINNERS HERE
});
```

This way we make sure we will always remove spinners at the end, even if we add another possible failure case in the fetchCoverage function.

Reply:
good to know, thx